integration_test/third_party_apps_data/agent/ops-agent/linux/enable_zookeeper

```diff
@@ -26,6 +26,17 @@ metrics:
       zookeeper:
         receivers:
           - zookeeper
+
+logging:
```

Comment:
Can we keep the receiver / pipeline IDs the same as the receiver type in the bare minimum config?

See more context in the description of https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764.